OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -184,14 +184,20 @@ void RegisterSettingsPanel(PanelType type, Func<Widget, Action> init, Func<Widge
 			resetPanelActions.Add(type, reset(panel));
 		}
 
+		static readonly Dictionary<WorldViewport, string> ViewportSizeNames = new Dictionary<WorldViewport, string>()
+		{
+			{ WorldViewport.Close, "Close" },
```

Comment:
Close is ~classic game size
Medium is ~720p
Far is ~1080p
Native/Furthest is always a zoom of 1.

The "~" is because the game will try to pick a size that is near but not necessarily matching these so we don't unnecessarily degrade the visual quality at different aspect ratios.